client-core/src/transaction_builder/raw_transfer_transaction_builder.rs

```diff
@@ -1057,11 +1066,11 @@ mod raw_transfer_transaction_builder_tests {
 
         builder.add_input((
             TxoPointer::new(random(), 0),
-            TxOut::new(transfer_addr.clone(), Coin::new(100).unwrap()),
+            TxOut::new(transfer_addr.clone(), Coin::new(500).unwrap()),
         ));
         builder.add_input((
             TxoPointer::new(random(), 0),
-            TxOut::new(transfer_addr, Coin::new(200).unwrap()),
+            TxOut::new(transfer_addr, Coin::new(500).unwrap()),
```

Comment:
why are the amount changes needed in tests?

Reply:
the fee was incorrect?

Reply:
the old one inputs value equals to outputs value,  after I add fee check, some tests will fail.